docs/how-to/ccv0.sh

```diff
@@ -78,31 +78,34 @@ Overview:
-- open_kata_console:            Stream the kata runtime's console
-- open_kata_shell:              Open a shell into the kata runtime
-- agent_pull_image:             Run PullImage command against the agent with agent-ctl
-- shim_pull_image:              Run PullImage command against the shim with ctr
-- agent_create_container:       Run CreateContainer command against the agent with agent-ctl
-- test:                         Test using kata with containerd
-- test_capture_logs:            Test using kata with containerd and capture the logs in the user's home directory
+- help:                             Display this help
+- all:                              Build and install everything, test kata with containerd and capture the logs
+- build_and_install_all:            Build and install everything
+- initialize:                       Install dependencies and check out kata-containers source
+- rebuild_and_install_kata:         Rebuild the kata runtime and agent and build and install the image
+- build_kata_runtime:               Build and install the kata runtime
+- configure:                        Configure Kata to use rootfs and enable debug
+- create_rootfs:                    Create a local rootfs
+- build_and_add_agent_to_rootfs:    Builds the kata-agent and adds it to the rootfs
+- build_and_install_rootfs:         Builds and installs the rootfs image
+- install_guest_kernel:             Setup, build and install the guest kernel
+- build_qemu:                       Checkout, patch, build and install QEMU
+- init_kubernetes:                  initialize a Kubernetes cluster on this system
+- crictl_create_cc_pod              Use crictl to create a new kata cc pod
+- crictl_create_cc_container        Use crictl to create a new busybox container in the kata cc pod
+- crictl_delete_cc                  Use crictl to delete the kata cc pod sandbox and container in it
+- kubernetes_create_cc_pod:         Create a Kata CC runtime busybox-based pod in Kubernetes
+- kubernetes_delete_cc_pod:         Delete the Kata CC runtime busybox-based pod in Kubernetes
+- kubernetes_create_ssh_demo_pod:   Create a Kata CC runtime pod based on the ssh demo
+- connect_to_ssh_demo_pod:          Ssh into the ssh demo pod, showing that the decryption succeeded
+- kubernetes_delete_ssh_demo_pod:   Delete the Kata CC runtime pod based on the ssh demo
+- open_kata_console:                Stream the kata runtime's console
+- open_kata_shell:                  Open a shell into the kata runtime
+- agent_pull_image:                 Run PullImage command against the agent with agent-ctl
+- shim_pull_image:                  Run PullImage command against the shim with ctr
+- agent_create_container:           Run CreateContainer command against the agent with agent-ctl
+- test:                             Test using kata with containerd
+- test_capture_logs:                Test using kata with containerd and capture the logs in the user's home directory
```

Comment:
Please could you sort this list alphabetically?

Reply:
I've added https://github.com/kata-containers/kata-containers/pull/3638/commits/f2243ffe34bb28a2d37b4c40afd68e1edada0dc8 in which I've sorted all the instructions. I'm happy to squash it into the main commit if you'd prefer?